### PR TITLE
Locked_items_right_section

### PR DESCRIPTION
**Pull request by MaksimDrobchak**
This PR:
- add input property isLocked(item) for definition whether item is locked
- by defautl isLocked = item => item.disabled
- support locked items in destination list by using this property
- make existing logic in source list to use isLocked(item) property as well

### CODE REVIEW

**Comment by stepovat** [comment]
Use isLocked  function everywhere you need item.disabled check

File: src/components/items/selected_item.js [hunk 1]
```diff
@@ -3,16 +3,22 @@ import PropTypes from "prop-types";
...
 
-const SelectedItem = ({ item, height, group }) => (
+const SelectedItem = ({ item, height, group, isLocked }) => (
   <div
-    className={group ? styles.with_grouping : styles.selected_item}
+    className={classnames({
+      [styles.with_grouping]: group,
+      [styles.selected_item]: !group,
+      [itemStyles.disabled]: isLocked(item),
+      [itemStyles.selected]: isLocked(item)
```
Comment (stepovat) [inline src/components/items/selected_item.js]: are you sure we need this?.... it's selected item component, it is only for selected

**Comment by stepovat** [comment]
@MaksimDrobchak  One comment, everything else looks good for me so far. Please fix the comment as we spoke offline and finish with tests coverage. Also, please don't forget codeclimate

@talyak @AmirAsaraf @yuval-rikover @liorheber If you take a look as well on this point - it will be great, thanks

File: src/components/items/selected_item.js [hunk 2]
```diff
@@ -5,15 +5,13 @@ import IconButton from "@material-ui/core/IconButton";
...
-import itemStyles from "./item.scss";
 
 const SelectedItem = ({ item, height, group, isLocked }) => (
   <div
     className={classnames({
       [styles.with_grouping]: group,
       [styles.selected_item]: !group,
-      [itemStyles.disabled]: isLocked(item),
-      [itemStyles.selected]: isLocked(item)
+      [styles.disabled]: isLocked(item)
```
Comment (stepovat) [inline src/components/items/selected_item.js]: please check if SelectedItem is something that can be overriden by user. 
If yes - then we should do the job for him, at least calculate is item disabled on higher level and pass result to this renderer. Do not put this responsibility on user.

**Comment by MaksimDrobchak** [comment]
Hi @talyak, could you please help again with codeclimate? 
There are strange notifications again with "duplicated blocks", although they are variables from prop and state... 
Thanks in advance!

**Pull request merged**